build/Makefile.pf.AIX

```diff
@@ -19,7 +19,7 @@ C_WARN_FLAGS=-qformat -qhalt=w -qflag=w:w -qsuppress=1540-2864
 # Common compiler flags (sent to compile command and link commands)
 # (-qthreaded is implied by using the _r version of xlc++)
 #64bit: COMMONFLAGS=-q64 -qlanglvl=strict98 -qrtti -qpic
-COMMONFLAGS=-q32 -qlanglvl=strict98 -qrtti -qpic -qextchk
+COMMONFLAGS=-q32 -qlanglvl=strict98 -qrtti -qpic -qextchk -qlanglvl=anonstruct
```

Comment:
That was a little tricky to find documentation in (took three or four tries). We're not changing anything in our code, so I can only imagine that the new header file, `sys/cdli_entuser.lncent.h`, must declare an anonymous structure. Is this correct?

It's odd that a system header file would require compiler mode changes like this, but I guess it's not the first time I've seen this.